Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -79,32 +101,39 @@ def get_packs_support_level_label(file_paths: List[str]) -> str:
 
     Args:
         file_paths(str): file paths
+        external_pr_branch (str): the branch of the external PR.
 
     Returns:
         highest support level of the packs that were changed, empty string in case no packs were changed.
     """
-    changed_pack_dirs = set()
+    pack_dirs_to_check_support_labels = set()
```

Comment:
Rename `pack_dirs_to_check_support_labels` to either one of:

- `pack_dirs_to_check_support_levels`
- `pack_dirs_to_check_support_levels_labels`